src/thunderbots/software/CMakeLists.txt

```diff
@@ -93,6 +99,7 @@ add_dependencies(backend_input ${catkin_EXPORTED_TARGETS})
 target_link_libraries(backend_input
         ${catkin_LIBRARIES}
         ${PROTOBUF_LIBRARIES}
+        ${G3LOG}
```

Comment:
Small nit, but it may be a good idea to add this to something like a ${COMMON_LIBS} variable; that way it should save some work if we add more libraries that are required by all nodes in the future.

Reply:
I'm not going to worry about this until we add more libs. In the future if we ended up with a small number of common libs I might vouch for keeping them separate anyway for the sake of explicitness, but that's a discussion for later.

Reply:
Ok 👍